app/deploy/api/pysoltrace.py

```diff
@@ -1360,7 +1360,7 @@ def __load_dll(self):
             # print("Loaded win32")
             #pdll = CDLL(cwd + "/coretraced.dll") # for debugging
         elif sys.platform == 'darwin':
-            pdll = CDLL(cwd + "/coretrace.dylib")  # Never tested
+            pdll = CDLL(cwd + "/coretrace_api.dylib")  # Never tested
```

Comment:
I believe this was a change made in NREL/SolTrace/develop that I pulled in when I merged. This pull request did not require any changes to pysoltrace.py.